internal/app/coroutines/rejectPromise.go

```diff
@@ -51,14 +51,14 @@ func RejectPromise(metadata *metadata.Metadata, req *t_api.Request, res func(*t_
 			res(&t_api.Response{
 				Kind: t_api.RejectPromise,
 				RejectPromise: &t_api.RejectPromiseResponse{
-					Status: t_api.ResponseNotFound,
+					Status: t_api.StatusPromiseNotFound,
 				},
 			}, nil)
 		} else {
 			p, err := result.Records[0].Promise()
 			if err != nil {
 				slog.Error("failed to parse promise record", "record", result.Records[0], "err", err)
-				res(nil, err)
+				res(nil, t_api.NewResonateError(t_api.ErrAIOStoreSerializationFailure, "", err))
```

Comment:
should we put "failed to parse promise record" here?

Reply:
and for others with an empty string

Reply:
ah missed thanks !